packages/react-playground/src/App.js

```diff
@@ -37,7 +39,17 @@ class App extends React.Component {
     this.setState({ disabled: !this.state.disabled });
   }
 
+  setSize(event) {
+    this.setState({ size: event.target.value });
```

Comment:
That's the browser interface for accessing the value of a form element change event. Think this should be `event.target.detail.state.value`?